openedx/core/djangoapps/oauth_dispatch/docs/decisions/0008-use-asymmetric-jwts.rst

```diff
@@ -47,23 +47,10 @@ be the only service configured with the aymmetric keypair, including its Private
 All other OAuth Clients will be configured with only the Public key portion of the asymmetric key pair.
 This will ultimately replace all uses of "symmetric" keys for signing JWTs.
 
-"kid" Key Identifier
```

Comment:
Removing this block because of
```
Considering that we are doing verification
by looping through all the available public keys, the ``kid`` parameter is not
as important as it was before. But it is still a good practice to have it.
```

Reply:
Yeah, got your point.
I've brought back the old header in the doc. 
And updated this part as well
https://github.com/openedx/edx-platform/pull/32270/files#diff-5f5c0ae6e79da08673e8a1bf4ad3d8cdea340124f0cc6d6b77ca3493cce00199R154